cmd/fynedesk/main_linux.go

```diff
@@ -14,12 +14,12 @@ import (
 
 func setupDesktop(a fyne.App) desktop.Desktop {
 	icons := internal.NewFDOIconProvider()
-	mgr, err := wm.NewX11WindowManager(a)
+	mgr, screensProvider, err := wm.NewX11WindowManager(a)
```

Comment:
Given that you have created a new constructor, NewX11ScreensProvider, why does this have to be returned by a different constructor?

Reply:
Fixed.